onnxruntime/test/python/onnxruntime_test_ort_trainer_with_mixed_precision.py

```diff
@@ -10,39 +10,45 @@
 
 class TestOrtTrainer(unittest.TestCase):
     def testBertTrainingMixedPrecision(self):
-        expected_losses = [11.0234375, 11.09375, 11.0078125, 11.0625, 11.03125, 11.0390625, 11.046875, 10.9921875]
-        expected_all_finites = [False, True, True, True, True, True, True, True]
-        expected_eval_loss = [10.960938]
+        expected_losses = [
+            11.034248352050781, 11.125300407409668, 11.006105422973633, 11.047048568725586,
+            11.027417182922363, 11.015759468078613, 11.060905456542969, 10.971782684326172]
+        expected_all_finites = [True, True, True, True, True, True, True, True]
```

Comment:
Is the change of `expected_all_finites` values also expected? The first all_finite was `False` now `True`.